rebasehelper/application.py

```diff
@@ -98,6 +98,8 @@ def __init__(self, cli_conf, execution_dir, results_dir, debug_log_file):
 
         self.kwargs['changelog_entry'] = self.conf.changelog_entry
 
+        self.kwargs['replace_old_version_with_macro'] = self.conf.replace_old_version_with_macro
```

Comment:
Having to do this defeats the purpose of defining options in plugins. `self.kwargs` should be populated with `self.conf` attibutes automatically, or `self.conf` should be passed to plugins.